custom_modules/MongoDbController/MongoControllerHelpers.js

```diff
@@ -53,19 +53,19 @@ class MongoControllerHelpers
                 const errResults = new MongoResults({ error: err, status: 500 });
                 reject(errResults);
             });
-		});
-	}
+        });
```

Comment:
Had a bunch of tabs in Mongo-related files. Replaced them with whitespace, as that's what VSCode prefers